dist/stat_data.py

```diff
@@ -800,6 +821,34 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_update_bytes', 'update key and value bytes', 'size'),
     CursorStat('cursor_update_bytes_changed', 'update value size change', 'size'),
 
+    ##########################################
+    # History statistics
+    ##########################################
+    HistoryStat('hs_gc_pages_evict', 'history pages added for eviction during garbage collection'),
+    HistoryStat('hs_gc_pages_removed', 'history pages removed for garbage collection'),
+    HistoryStat('hs_gc_pages_visited', 'history pages visited for garbage collection'),
+    HistoryStat('hs_write_addr_prepared', 'page addresses written that included at least one prepare state'),
+    HistoryStat('hs_write_addr_newest_start_durable_ts', 'page addresses written that include a newest start durable timestamp '),
+    HistoryStat('hs_write_addr_newest_stop_durable_ts', 'page addresses written that include a newest stop durable timestamp '),
+    HistoryStat('hs_write_addr_oldest_start_ts', 'page addresses written that include a oldest start timestamp '),
+    HistoryStat('hs_write_addr_oldest_start_txn', 'page addresses written that include a oldest start transaction ID '),
+    HistoryStat('hs_write_addr_newest_stop_ts', 'page addresses written that include a newest stop timestamp '),
+    HistoryStat('hs_write_addr_newest_stop_txn', 'page addresses written that include a newest stop transaction ID'),
+    HistoryStat('hs_write_pages_prepared', 'pages written that included at least one prepare state'),
+    HistoryStat('hs_write_pages_durable_start_ts', 'pages written that included at least one start durable timestamp'),
+    HistoryStat('hs_write_pages_durable_stop_ts', 'pages written that included at least one stop durable timestamp'),
+    HistoryStat('hs_write_pages_start_ts', 'pages written that included at least one start timestamp'),
+    HistoryStat('hs_write_pages_start_txn', 'pages written that included at least one start transaction ID'),
+    HistoryStat('hs_write_pages_stop_ts', 'pages written that included at least one stop timestamp'),
+    HistoryStat('hs_write_pages_stop_txn', 'pages written that included at least one stop transaction ID'),
+    HistoryStat('hs_write_rec_prepared', 'records written that included a prepare state'),
+    HistoryStat('hs_write_rec_durable_start_ts', 'records written that included a start durable timestamp'),
+    HistoryStat('hs_write_rec_durable_stop_ts', 'records written that included a stop durable timestamp'),
+    HistoryStat('hs_write_rec_start_ts', 'records written that included a start timestamp'),
+    HistoryStat('hs_write_rec_start_txn', 'records written that included a start transaction ID'),
+    HistoryStat('hs_write_rec_stop_ts', 'records written that included a stop timestamp'),
+    HistoryStat('hs_write_rec_stop_txn', 'records written that included a stop transaction ID'),
```

Comment:
This is a lot of new statistics. I was trying to think if internal statistics are interesting here, be interested to hear the use case you have in mind? I can see leaf-page counts, but I'm not sure when I'd want to see internal page counts.